src/tools/plan-generator.ts

```diff
@@ -116,11 +117,16 @@ export class PlanGenerator {
       return ktUtils.applyFun('StorageKeyParser.parse', [quote(handle.storageKey.toString())]);
     }
     if (handle.fate === 'create') {
-      return ktUtils.applyFun('CreateableStorageKey', [quote(handle.id)]);
+      return ktUtils.applyFun('CreateableStorageKey', [quote(handle.id || this.createHandleName())]);
```

Comment:
id || name is confusing, maybe rename the method to `createRandomId` or smth more descriptive of what it's really doing.

Reply:
Thanks, that's a better name.